### PR TITLE
Ad 14/bulletin board

### PR DESCRIPTION
**Pull request by adambudziak**
Initial implementation

### CODE REVIEW

File: backend/apollo/elections/models/election.py [hunk 1]
```diff
@@ -87,6 +96,14 @@ def close(self) -> None:
     def state_string(self) -> str:
         return str(Election.State.choices[self.state])
 
+    @property
+    def seed_hash(self) -> bytes:
+        if self.seed is None:
+            raise ValueError("Voter seed is None")
```
Comment (KonradSwierczynski) [inline backend/apollo/elections/models/election.py:102]: Should be `"Election seed is None"`

File: backend/poetry.lock [hunk 2]
```diff
@@ -25,7 +25,7 @@ version = "1.4.4"
 [[package]]
 category = "dev"
 description = "Disable App Nap on OS X 10.9"
-marker = "sys_platform == \"darwin\""
+marker = "python_version >= \"3.4\" and sys_platform == \"darwin\" or sys_platform == \"darwin\""
```
Comment (KonradSwierczynski) [inline backend/poetry.lock:28]: weird change, but I guess it auto-generated

Comment (adambudziak) [reply backend/poetry.lock:28]: Wow, look what github  added!
```suggestion
wow a new feature = "python_version >= \"3.4\" and sys_platform == \"darwin\" or sys_platform == \"darwin\""
```

Comment (KonradSwierczynski) [reply backend/poetry.lock:28]: I saw it on gitlab years ago...
But it's cool every time since then

**Pull request merged**